### ISSUE TITLE
Where do you download gradle-wrapper.jar?

### ISSUE THREAD

**Issue by cpburnz** [gradle/gradle#30393]
### Issue type

Missing information

### Problem description

Gradle binary-only and complete releases can be downloaded from [here](https://gradle.org/releases/). But where do you download the `gradle-wrapper.jar` for each release? Checksums are published [here](https://gradle.org/release-checksums/) which includes the checksum for the wrapper JAR.

### Page with the problem

https://gradle.org/releases/

**Comment by ov7a** [comment]
Your issue appears to be a question about how to use Gradle. They should be asked in a different channel, so we will close it. Your question might be answered in Gradle documentation or on our community Slack or forums. Find links to these resources on [help.gradle.org](https://help.gradle.org/).

If you think our analysis is wrong, please provide us with more detailed information explaining why.

***

Please refer to the docs:
* https://docs.gradle.org/current/userguide/gradle_wrapper.html#gradle_wrapper_reference
* https://docs.gradle.org/current/userguide/installation.html

In short, we don't provide wrapper as a downloadable artifact, it is generated by Gradle.
